feature/bgp/Import_Export_Policy/otg_tests/README.md

```diff
@@ -0,0 +1,216 @@
+* Access to configure the DUT and ATE.
+* Understanding of BGP attributes (AS-Path, communities, local preference, MED).
+
+## Test Case Scenario
+
+This test case will focus on two scenarios:
+
+* Export Policy: Router exports only specific prefixes to ATE based on a policy.
+* Import Policy: Router imports only specific prefixes from ATE based on a policy.
+
+## Test Steps
+
+### Initial Setup (Pre-Test)
+
+### DUT Configuration (Basic BGP Peering):
+
+* Configure interface IP addresses (DUT: IPV4 - 10.1.1.0/31 & IPV6 - 2001:db8:204:114::0/127 , ATE: IPV4 - 10.1.1.1/31 & IPV6 - 2001:db8:204:114::1/127 ).
+* Enable BGP on the DUT.
+* Establish an EBGP peering session with the ATE (using ATE's AS as  AS65002).
+* Ensure the DUT has some local routes (loopback interfaces advertised into BGP) that can be filtered.
+
+
+### ATE Configuration (Basic BGP Emulation):
+
+* Configure the ATE port with an IP address (10.1.1.1/31) & IPv6 address (2001:db8:204:114::1/127).
+* Create a BGP Emulated Router on the ATE, acting as AS 65002.
+* Establish a BGP peering session with the DUT's IP (10.1.1.0) & IPV6 (2607:f8b0:8007:614f::).
+
+* Advertise multiple unique BGP routes from ATE, some intended to be filtered, and some to be allowed.
+  * Allowed routes: IPV4 192.0.2.1/32 (AS-Path: 65002), 192.0.2.2/32 (AS-Path: 65002) and IPV6 2001:db8:300:100::0/127 (AS-Path: 65002) , 2001:db8:300:101::0/127 (AS-Path: 65002)
+  * Denied routes:  IPV4 198.51.100.1/32 (AS-Path: 65002 65003), 198.51.100.2/32 (AS-Path: 65002 65004) and IPV6 2001:db8:400:100::1/128 (AS-Path: 65002 65004) , 2001:db8:400:101::1/128 (AS-Path: 65002 65004)
+
+### Verify BGP Peering:
+
+* On DUT: Check the BGP summary
+* On ATE: Check BGP session status in IxNetwork.
+* Ensure both the DUT and ATE have exchanged routes without any policy applied yet.
+
+### Test Export Policy (Prefix-list based)
+
+Objective: Only allow local routes 172.16.1.0/24 and 172.16.2.0/24 & 2001:db8:250:110::/64 and 2001:db8:251:110::/64  to be advertised from DUT to ATE. 192.168.10.0/24 & 2001:db8:299:110::/64 should be denied.
+
+Configure Export Policy on DUT:
+Create a prefix-list to match the desired prefixes.
+Create a route-map/policy-statement to apply the prefix-list.
+Apply the route-map/policy-statement to the BGP neighbor 10.1.1.1 & 2607:f8b0:8007:614f::1 as an out policy.
+
+
+### Clear BGP Session (Optional, but good practice):
+* On DUT: clear ip & ipv6 bgp sessions towards 10.1.1.1 & 2607:f8b0:8007:614f::1 out (or soft reset)
+* This forces the DUT to re-advertise routes based on the new policy.
+
+### Verify Export Policy:
+
+* ATE:
+  * Go to the "Routes" tab for the emulated BGP router.
+  * Observe the "Learned Routes" from the DUT.
+  * Expected: Only 172.16.1.0/24 and 172.16.2.0/24 & 2001:db8:250:110::/64 and 2001:db8:251:110::/64 should be learned.
+  * Expected: 192.168.10.0/24 & 2001:db8:299:110::/64 should NOT be learned.
```

Comment:
This seems very human-centric. Can you please rephrase based on the case that you want -- i.e., what is it you're expecting to see? (the expected + expected bullets are much more generic.)